extensions/amp-analytics/0.1/vendors/gtag.js

```diff
@@ -22,6 +22,12 @@ export const GTAG_CONFIG = /** @type {!JsonObject} */ ({
         'dr': 'DOCUMENT_REFERRER',
         'dl': 'SOURCE_URL',
       },
+      'conversion': {
+        'gclsrc': 'QUERY_PARAM(gclsrc)',
+        'hasGcl': '$IF(QUERY_PARAM(gclid, 1, 0))',
```

Comment:
Looks like the parenthesis is still in the wrong place :(

Reply:
ok, it's time to get some coffee